tests/integration/targets/s3_object/tasks/get_object_info.yml

```diff
@@ -0,0 +1,123 @@
+---
+- name: Integration test for s3_object_info
+  block:
+    - name: Create a simple s3_bucket
+      s3_bucket:
+        name: "{{ bucket_name }}-object-info-test"
+        state: present
+        object_ownership: BucketOwnerEnforced
+        tags:
+          "lowercase spaced": "hello cruel world"
+          "Title Case": "Hello Cruel World"
+          CamelCase: "SimpleCamelCase"
+          snake_case: "simple_snake_case"
+
+    - name: check that temp directory was made
+      assert:
+        that:
+        - remote_tmp_dir is defined
+
+    - name: Create content
+      set_fact:
+          content: "{{ lookup('password', '/dev/null chars=ascii_letters,digits,hexdigits,punctuation') }}"
+
+    - name: Create local object-info-test.txt
+      copy:
+        content: "{{ content }}"
+        dest: "{{ remote_tmp_dir }}/object-info-test.txt"
+
+    - name: stat the file
+      stat:
+        path: "{{ remote_tmp_dir }}/object-info-test.txt"
+        get_checksum: yes
+
+    - name: Upload a object to bucket
+      amazon.aws.aws_s3:
+        bucket: "{{ bucket_name }}-object-info-test"
+        object: object-info-test.txt
+        src: "{{ remote_tmp_dir }}/object-info-test.txt"
+        mode: put
+        permission: bucket-owner-full-control
+        tags:
+          "lowercase spaced": "hello cruel world"
+          "Title Case": "Hello Cruel World"
+          CamelCase: "SimpleCamelCase"
+          snake_case: "simple_snake_case"
+
+    - name: List keys
+      amazon.aws.s3_object:
+        bucket: "{{ bucket_name }}-object-info-test"
+        mode: list
+      register: list_keys_result
+
+    - name: Get s3 object info
+      amazon.aws.s3_object_info:
+        bucket_name: "{{ bucket_name }}-object-info-test"
+        object_name: "{{ list_keys_result.s3_keys[0] }}"
+      register: info_result
+    - debug:
+        var: info_result
+
+    - assert:
+        that:
+          - info_result is not failed
+          - info_result is not changed
+          - '"object_info" in info_result'
+          - info_result.object_info | length != 0
+
+    - name: Get s3 object info
```

Comment:
I didn't dig into, but I'm just asking for further info. This is the log of this tasks

```
"object_info": {
2294	2022-08-24 21:30:03.591254 | controller |         "object_acl": {
2295	2022-08-24 21:30:03.591258 | controller |             "msg": "Object ACL not found, are you sure the bucket/object configuration exist?"
2296	2022-08-24 21:30:03.591262 | controller |         },
2297	2022-08-24 21:30:03.591266 | controller |         "object_attributes": {
2298	2022-08-24 21:30:03.591270 | controller |             "msg": "Object attributes not found, are you sure the bucket/object configuration exist?"
2299	2022-08-24 21:30:03.591275 | controller |         },
2300	2022-08-24 21:30:03.591278 | controller |         "object_lock_configuration": {
2301	2022-08-24 21:30:03.591282 | controller |             "msg": "Object lock configuration not found, are you sure the bucket/object configuration exist?"
2302	2022-08-24 21:30:03.591287 | controller |         },
2303	2022-08-24 21:30:03.591291 | controller |         "object_tagging": {
2304	2022-08-24 21:30:03.591294 | controller |             "CamelCase": "SimpleCamelCase",
2305	2022-08-24 21:30:03.591298 | controller |             "Title Case": "Hello Cruel World",
2306	2022-08-24 21:30:03.591302 | controller |             "lowercase spaced": "hello cruel world",
2307	2022-08-24 21:30:03.591306 | controller |             "snake_case": "simple_snake_case"
2308	2022-08-24 21:30:03.591310 | controller |         }
2309	2022-08-24 21:30:03.591314 | controller |     },
```

Since in the log returned by the previous task https://github.com/ansible-collections/amazon.aws/pull/977/files#diff-3b47451f92053737894c5813b0701a668c1ec5f6feb02e78ce3b7b11770f6a2fR53 e_tag had a value ("e_tag": "\"00692ee0f2460b4f4f582f0b0f91a6d9\"",), shouldn't we expect to have it as an attribute as well?